Source/Scene/Model.js

```diff
@@ -399,6 +403,33 @@ define([
         this.show = defaultValue(options.show, true);
 
         /**
+         * Determines if the model will be highlighted.
+         *
+         * @type {Boolean}
+         *
+         * @default false
+         */
+        this.highlight = defaultValue(options.highlight, false);
+
+        /**
+         * The highlight color
+         *
+         * @type {Color}
+         *
+         * @default Color()
+         */
+        this.highlightColor = defaultValue(options.highlightColor, new Color());
```

Comment:
Minor, but use the ternary operator to avoid the allocation when it is not needed.